index.html

```diff
@@ -0,0 +1,128 @@
+        <div class="skill-item">
+          <h3>CSS</h3>
+          <p>Proficient</p>
+          <i class="fab fa-css3-alt"></i>
+        </div>
+      </div>
+    </section>
+    <hr>
+    <section class="about">
+      <div class="main">
+        <div class="about-content">
+          <img src="images/Screenshot (19).png" />
+          <div class="about-text">
+            <h2 id="Project">About Projects</h2>
+            <h5>Linters <span class="red">Config</span></h5>
+            <p>Linters are tools that analyze code for potential errors, bugs, stylistic inconsistencies, and other issues.</p>
+            <a href="https://zainabbehzad.github.io/First_project/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button type="button">Source</button></a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="projects" class="projects">
+      <h2 id="Projects">Some of my Projects</h2>
+      <div class="project-grid">
+        <div class="project-item">
+          <img class="project-img" src="./images/Screenshot (15).png">
+          <div class="buttons">
+            <a href="https://zainabbehzad.github.io/survey-form/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button class="button">Source</button></a>
+          </div>
+        </div>
+        <div class="project-item">
+          <img class="project-img" src="./images/Screenshot (17).png">
+          <div class="buttons">
+            <a href="https://zainabbehzad.github.io/Tribute-page/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button class="button">Source</button></a>
+          </div>
+        </div>
+        <div class="project-item">
+          <img class="project-img" src="./images/Screenshot (18).png">
+          <div class="buttons">
+            <a href="https://zainabbehzad.github.io/technical-documentation-page/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button class="button">Source</button></a>
+          </div>
+        </div>
+        <div class="project-item">
+          <img class="project-img" src="./images/Screenshot (16).png">
+          <div class="buttons">
+            <a href="https://zainabbehzad.github.io/Product-Landing-Page/"><button class="button">Live</button></a>
+            <a href="https://github.com/zainabbehzad"><button class="button">Source</button></a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div  class="contact-me">
+      <p id="Contact">A Beautiful Website.</p>
+      <a class="button-two" href="#">Hire Me</a>
+    </div>
+    <footer>
+      <P>Zainab Behzad</P>
+      <p>For more HTML , CSS, and coding tutorial - please click on the link below to subscribe to my channel:</p>
+      <div class="profile-link">
+        <a href="#" target="_blank"><i class="fa-brands fa-facebook"></i></a>
```

Comment:
- Add your facebook link here; if you don't have an FB account or don't want to share, remove it.